packages/config/src/json/tiff.config.ts

```diff
@@ -78,7 +78,7 @@ async function loadStacFromPath(target: string): Promise<StacCollection | null>
  *
  * @returns Imagery configuration generated from the path
  */
-export async function imageryFromTiffPath(target: string, Q: pLimit.Limit): Promise<ConfigImagery> {
+export async function imageryFromTiffPath(target: string | string, Q: pLimit.Limit): Promise<ConfigImagery> {
```

Comment:
What is `string | string` DateType? I never see this before.

Reply:
I wonder how that got there! `string | string` is just a `string`!